tests/tests_pytorch/trainer/optimization/test_optimizers.py

```diff
@@ -735,15 +729,10 @@ def configure_optimizers(self):
         limit_train_batches=limit_train_batches,
         limit_val_batches=0,
     )
-
-    with patch.object(CustomEpochScheduler, "step") as mock_method_epoch, patch.object(
-        torch.optim.lr_scheduler.StepLR, "step"
-    ) as mock_method_step:
+    with mock.patch.object(CustomEpochScheduler, "step") as mock_method_epoch:
         trainer.fit(model)
 
     assert mock_method_epoch.mock_calls == [call(epoch=e) for e in range(max_epochs)]
-    # first step is called by PyTorch LRScheduler
-    assert mock_method_step.call_count == max_epochs * limit_train_batches + 1
```

Comment:
can we keep the step-level scheduler check? Maybe parametrizing this test or writing a duplicate that sets the scheduler on a step interval

Reply:
Actually, the simplest seems to be to just keep all assertions but reduce the number of optimizers to 1.